src/components/Input/SFloatInput.vue

```diff
@@ -36,11 +40,31 @@ const decimalsValidator = x => x === undefined || x >= 0
 })
 export default class SFloatInput extends Vue {
   @Prop({ type: String, default: DEFAULT_VALUE }) readonly value!: string
-  @Prop({ type: String, default: '0.0' }) readonly placeholder!: string
+  @Prop({ type: String }) readonly placeholder!: string
+  @Prop({ type: Object, default: () => {} }) readonly delimiters?: object
   @Prop({ type: Number, default: undefined, validator: decimalsValidator }) readonly decimals!: number
   @Prop({ type: [String, Number], default: undefined, validator: isNumberLikeValue }) readonly max!: string | number
 
+  get placeholderValue (): string {
+    return this.placeholder || '0'.concat(this.delimitersConfig.decimal, '0')
+  }
+
+  get delimitersConfig (): any {
+    return this.delimiters || DEFAULT_DELIMITERS
```

Comment:
This property will be a redudant after `delimeters` props definition update (like in my comment) , please use `this.delimeters`